Casks/g/gpgfrontend.rb

```diff
@@ -1,14 +1,14 @@
 cask "gpgfrontend" do
-  version "2.1.2"
+  version "2.1.3"
 
   on_big_sur do
```

Comment:
```suggestion
  on big_sur :or_older do

```

Unrelated to this PR, but makes sure that the Caskfile is readable on older OS.